src/NuGet.Clients/NuGet.PackageManagement.UI/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "VSTHRD003:Avoid awaiting foreign Tasks", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.PackageLoadContext.GetInstalledPackagesAsync~System.Threading.Tasks.Task{NuGet.PackageManagement.VisualStudio.PackageCollection}")]
```

Comment:
If there are fewer than 10 `VSTHRD003` warnings in the project, I'd prefer to use `#pragma disable ...` in the code, rather than suppressing it for the whole project. That way we hopeless are less likely to introduce new code that would trigger the warning.

Reply:
This doesn't disable it for the whole class, but rather for the specific method. 

Specifically, see: 

> Target = "~M:NuGet.PackageManagement.UI.PackageLoadContext.GetInstalledPackagesAsync~System.Threading.Tasks.Task{NuGet.PackageManagement.VisualStudio.PackageCollection}"